gitcoins-cordapp/build.gradle

```diff
@@ -0,0 +1,144 @@
+buildscript {
+    ext {
+        corda_release_group = 'net.corda'
+        corda_release_version = '5.0-SNAPSHOT'
+        corda_gradle_plugins_version = '4.0.42'
+        kotlin_version = '1.2.71'
+        junit_version = '4.12'
+        quasar_version = '0.7.10'
+        spring_boot_version = '2.0.2.RELEASE'
+        spring_boot_gradle_plugin_version = '2.0.2.RELEASE'
+        slf4j_version = '1.7.25'
+        log4j_version = '2.11.2'
+        corda_platform_version = '4'
```

Comment:
Could we have `tokens_sdk='com.r3.tokens-sdk'`
`tokens_sdk_version='1.0-SNAPSHOT'`

and then you could replace all the places where it's hardcoded as a string?